marketplace/hooks.tsx

```diff
@@ -72,7 +73,9 @@ export const useCreateFractionalMakerAsk = ({
 
   const chainId = useChainId();
   const client = useHypercertClient();
-  const { address } = useAccount();
+  const { address: connectedAddress } = useAccount();
```

Comment:
Can you explain the reasoning for using the selected account here? If the user selects the Safe, it wouldn't work anyway as it needs custom logic to support off-chain message signing, or am I missing something?